src/sources/aws_s3/sqs.rs

```diff
@@ -457,7 +470,7 @@ impl IngestorProcess {
                 let aws_region = Bytes::from(s3_event.aws_region.as_str().as_bytes().to_vec());
 
                 let mut stream = lines.filter_map(move |line| {
-                    emit!(&SqsS3EventReceived {
+                    emit!(&SqsS3EventsReceived {
                         byte_size: line.len()
```

Comment:
This `byte_size` is inaccurate. It should be using `ByteSizeOf::size_of` to calculate the in-memory byte count.